Runtime/Scripts/GameplayReferences.cs

```diff
@@ -383,15 +361,60 @@ public static Transform HMD
             }
         }
 
-#endregion
+        #endregion
 
 #region Controllers
 
+        /// <summary>
+        ///  A refrence to the controller pointer (if) instantiated for exitpoll
+        /// </summary>
+        internal static GameObject controllerPointer;
+
+        public static bool SDKSupportsControllers
+        {
+            get
+            {
+                //hand tracking not currently part of the setup process. may work, but not implemented as fully as other SDKs
+#if C3D_MRTK
+                return false;
+#endif
+                return true;
+            }
+        }
+
+
         //dynamic objects set as controllers call 'set controller' on enable passing a reference to that transform. input device isn't guaranteed to be valid at this point
 
         static Transform[] controllerTransforms = new Transform[2];
         static InputDevice[] controllerDevices = new InputDevice[2];
 
+        public static float rightTriggerValue;
+        public static float leftTriggerValue;
+
+        //updates controller and hmd inputdevices to call events when states change
```

Comment:
Added option to retrieve trigger value from GameplayReferences. Will be helpful to have it one place.